firestore-bigquery-export/README.md

```diff
@@ -13,7 +13,7 @@ The extension creates and updates a [dataset](https://cloud.google.com/bigquery/
 
 If you create, update, delete, or import a document in the specified collection, this extension sends that update to BigQuery. You can then run queries on this mirrored dataset.
 
-Note that this extension only listens for _document_ changes in the collection, but not changes in any _subcollection_. You can, though, install additional instances of this extension to specifically listen to a subcollection or other collections in your database. Or if you have the same subcollection across documents in a given collection, you can use `{wildcard}` notation to listen to all those subcollections (for example: `chats/{chatid}/posts`).
+Note that this extension only listens for _document_ changes in the collection, but not changes in any _subcollection_ nested within. You can, though, install additional instances of this extension to specifically listen to a sub-collection or other collections in your database.
```

Comment:
1. I'd make it more clear only a specific subcollection
"You can, though, install additional instances of this extension to listen to a specific sub-collection or other collections in your database. You can only listen on one specific collection or subcollection at once (wildcard subcollections are not supported)."
2. Installing additional extensions was counterintuitive to me
"To install additional extensions, revisit the page (https://firebase.google.com/products/extensions) and add another instance of "Export Collections to BigQuery."

Reply:
Thanks for the suggestions. I have revised the `README.md` with some of your changes. It is likely it'll change further as it's not my decision on wording of documentation and such!